src/components/Movie.jsx

```diff
@@ -0,0 +1,72 @@
+import { useDispatch, useSelector } from 'react-redux'
+import starredSlice from '../data/starredSlice'
+import watchLaterSlice from '../data/watchLaterSlice'
+import placeholder from '../assets/not-found-500X750.jpeg'
+
+const Movie = ({ movie, viewTrailer, closeCard }) => {
+
+    const state = useSelector((state) => state)
+    const { starred, watchLater } = state
+    const { starMovie, unstarMovie } = starredSlice.actions
+    const { addToWatchLater, removeFromWatchLater } = watchLaterSlice.actions
+
+    const dispatch = useDispatch()
+
+    const myClickHandler = (e) => {
+        if (!e) var e = window.event
+        e.cancelBubble = true
+        if (e.stopPropagation) e.stopPropagation()
+        e.target.parentElement.parentElement.classList.remove('opened')
+    }
+
+    return (
+        <div className="wrapper col-3 col-sm-4 col-md-3 col-lg-3 col-xl-2">
+        <div className="card" onClick={(e) => e.currentTarget.classList.add('opened')} >
+            <div className="card-body text-center">
+                <div className="overlay" />
+                <div className="info_panel">
+                    <div className="overview">{movie.overview}</div>
+                    <div className="year">{movie.release_date?.substring(0, 4)}</div>
+                    {!starred.starredMovies.map(movie => movie.id).includes(movie.id) ? (
+                        <span className="btn-star" data-testid="starred-link" onClick={() => 
+                            dispatch(starMovie({
+                                id: movie.id, 
+                                overview: movie.overview, 
+                                release_date: movie.release_date?.substring(0, 4),
+                                poster_path: movie.poster_path,
+                                title: movie.title
+                            })
+                        )}>
+                            <i className="bi bi-star" />
+                        </span>
+                    ) : (
+                        <span className="btn-star" data-testid="unstar-link" onClick={() => dispatch(unstarMovie(movie))}>
+                            <i className="bi bi-star-fill" data-testid="star-fill" />
+                        </span>
+                    )}
+                    {!watchLater.watchLaterMovies.map(movie => movie.id).includes(movie.id) ? (
+                        <button type="button" data-testid="watch-later" className="btn btn-light btn-watch-later" onClick={() => dispatch(addToWatchLater({
+                                id: movie.id, 
+                                overview: movie.overview, 
+                                release_date: movie.release_date?.substring(0, 4),
+                                poster_path: movie.poster_path,
+                                title: movie.title
+                        }))}>Watch Later</button>
```

Comment:
Same suggestion about sending the movie data object
```suggestion
                        <button type="button" data-testid="watch-later" className="btn btn-light btn-watch-later" onClick={() => dispatch(addToWatchLater({ ...movie, release_date: movie.release_date?.substring(0, 4) }))}>Watch Later</button>
```